src/components/QuestionLists.module.scss

```diff
@@ -21,3 +21,20 @@
     display: flex;
     justify-content: center;
 }
+
+.black {
```

Comment:
Rename class to element where is used. `.black` class sounds for me as global css and not for certain component.

Reply:
Rename all the occurrences.